cmake/modules/platform/os/zos.cmake

```
@@ -43,7 +43,11 @@ list(APPEND CMAKE_INCLUDE_PATH "/usr/lpp/cbclib/include")
 # Create helper targets for specifying ascii/ebcdic options
 add_library(omr_ascii INTERFACE)
 target_compile_definitions(omr_ascii INTERFACE -DIBM_ATOE)
-target_compile_options(omr_ascii INTERFACE "-Wc,convlit(ISO8859-1),nose,se(${CMAKE_CURRENT_LIST_DIR}/../../../../util/a2e/headers)")
+if(CMAKE_C_COMPILER_IS_OPENXL)
+	target_compile_options(omr_ascii INTERFACE -fexec-charset=ISO8859-1 -isystem ${CMAKE_CURRENT_LIST_DIR}/../../../../util/a2e/headers)
```

match the formatting on line 49 for consistency
```suggestion
	target_compile_options(omr_ascii INTERFACE "-fexec-charset=ISO8859-1 -isystem ${CMAKE_CURRENT_LIST_DIR}/../../../../util/a2e/headers")
```

---

[  4%] Building C object runtime/omr/thread/CMakeFiles/j9thr_obj.dir/common/j9sem.c.o
ibm-clang: error: invalid value 'ISO8859-1 -isystem .../omr/cmake/modules/platform/os/../../../../util/a2e/headers' in '-fexec-charset=ISO8859-1 -isystem .../omr/cmake/modules/platform/os/../../../../util/a2e/headers'

This produces an error, as it treats the whole string as a argument, I might have to leave it as it is to proceed.